### DISCUSSION TITLE
Acceso a Backend?

### DISCUSSION THREAD

**Discussion by iguelAL** [Launch-X-Latam/MisionFrontEnd#184] [Q&A]
Hola, saben si ya se puede unir al grupo de backend?, Hace un momento en admin del grupo de teams mando un formulario para unirse al grupo de BackEnd, estoy un poco confundió jaja.

**Comment by RaymundoSantorski** [comment]
Ya se puede, hasta alcanzar el limite de solicitudes diario, es para que ya todos estemos dentro cuando inicie la misión

**Comment by Tongas5625** [comment]
Buenos dias, serias tan amable de compartir el link del formulario por favor

**Reply by JoseEduardoVelazquezCN** [reply]
Está en el grupo de teams, en el apartado de "avisos Github", ahí está toda la información que necesitas
